java/springboot_hikaricp_using_dsql_connector/src/main/resources/application.yml

```diff
@@ -0,0 +1,19 @@
+spring:
+  datasource:
+    url: jdbc:aws-dsql:postgresql://${CLUSTER_ENDPOINT}/postgres
+    username: ${CLUSTER_USER}
+    driver-class-name: software.amazon.dsql.jdbc.DSQLConnector
+    hikari:
+      data-source-properties:
+        token-duration-secs: 28800  # 8 hours
+        profile: ${PROFILE}        # Optional: AWS profile for authentication
+      max-lifetime: 3000000        # 50 minutes (shorter than Aurora DSQL's 60-minute limit)
+      minimum-idle: 2
+      maximum-pool-size: 10
+      idle-timeout: 300000         # 5 minutes
+      connection-timeout: 30000    # 30 seconds
+      validation-timeout: 5000     # 5 seconds
+      connection-test-query: "SELECT 1"
+logging:
+  level:
+    software.amazon.dsql: DEBUG
```

Comment:
How does this interact with the `software.amazon.dsql.level` property in the `logging.properties` file? Do they control different things or is only one of these necessary?